### ISSUE TITLE
Unable to create or edit content

### ISSUE THREAD

**Issue by hishamco** [OrchardCMS/OrchardCore#3714]
After PR #3626, I'm not unable to create or edit content and this error show up: 'ZoneShapes' does not contain a definition for 'HarvestAndSortTabs'

Did I miss something?

/cc @deanmarcussen

**Comment by deanmarcussen** [comment]
I can't repro.

Do you need to rebuild Contents Module?
`HarvestAndSortTabs` is a static helper in `ZoneShapes.cs`

Can you screenshot if still having error?

**Comment by hishamco** [comment]
Do I need to setup a site again?

**Comment by hishamco** [comment]
![Exception](https://user-images.githubusercontent.com/3237266/58483429-f811aa80-8168-11e9-91c8-cebca4692f96.png)

**Comment by deanmarcussen** [comment]
Have you done a full rebuild of the source code? 

If it's showing up in intellisense then it sounds like the dll just hasn't been compiled.

Also try recycling IIS if you're using it to host your site.
I see the IIS dotnet core developer module locking dll's with Build with VS 2019 a lot

**Comment by hishamco** [comment]
I'miss using VS 2017, let me try again ..

**Comment by hishamco** [comment]
Seems it was a locking issue with IIS Express
